source/libs/api.js

```diff
@@ -1,11 +1,19 @@
+import OptionsSync from 'webext-options-sync';
+
 const cache = new Map();
+const headers = new Headers();
+const options = new OptionsSync().getAll();
 
 export default async endpoint => {
 	if (cache.has(endpoint)) {
 		return cache.get(endpoint);
 	}
+	const {personalToken = ''} = await options;
+	if (personalToken.trim().length === 40) {
```

Comment:
I don't think it should just silently not set the header if this is not true. It should instead throw if this is not true.

Also make `40` a descriptive constant variable instead, for readability.

Reply:
Should it throw an actual error or log an error message to the console? Or both?

Reply:
Actually, we could instead use the `pattern` attribute on the `input` to prevent whitespace, so this check would be moot.